### DISCUSSION TITLE
Problemi con riavvio servizio Home Assistant

### DISCUSSION THREAD

**Discussion by braudoo** [indomus/forum#1537] [HUB: Home Assistant]
Salve a tutti, per qualche strana ragione ogni volta che riavvio il server di Home Assistant, il servizio non riparte in automatico.
Per far ripartire il servizio devo lanciare il comando di riavvio.
Facendo un pò di ricerche sembra che il problema sia il fatto che non trovi la direcotry `hass`.

```
pi@raspberrypi:~ $ sudo systemctl status home-assistant@homeassistant
● home-assistant@homeassistant.service - Home Assistant
   Loaded: loaded (/etc/systemd/system/home-assistant@homeassistant.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2022-02-24 18:57:17 GMT; 36min ago
  Process: 17594 ExecStart=/srv/homeassistant/bin/hass -c /home/homeassistant/.homeassistant (code=exited, status=100)
 Main PID: 17594 (code=exited, status=100)
```
Andando fino **/srv/homeassistant/bin** la cartella hass non la trovo.

Qualcuno riuscirebbe ad aiutarmi?

Grazie.

**Comment by KC-inDomus** [comment]
devi aggiornare il lancio del servizio: https://indomus.it/news/hot-aggiornamento-2022-2-per-home-assistant-core/?ht-comment-id=4162463

**Reply by braudoo** [reply]
Fantastico.
Grazie mille.
